docs/source/_templates/versioning.html

```diff
@@ -0,0 +1,46 @@
+{% if versions %}
+
+<!-- Start of versioning -->
+
+<div class="sidebar-tree">
+  <p class="caption" role="heading">
+    <span class="caption-text">{{ _('Versions') }}</span>
+  </p>
+  <ul>
+    <li class="toctree-l1">
+      <a
+        id="version_number"
+        class="reference internal"
+        href="../stable/index.html"
+        >stable</a
+      >
+    </li>
+    <li class="toctree-l1">
+      <a id="commit_hash" class="reference internal" href="../master/index.html"
+        >developer</a
```

Comment:
Can we rename `developer` to `master` (to match the branch name)?

Reply:
Can do. should the version warning banner be changed too:

![image](https://user-images.githubusercontent.com/90272361/155854689-bebc5e6f-c84a-440e-8b2e-8f3e5c3c5577.png)

Reply:
done!

![image](https://user-images.githubusercontent.com/90272361/155888656-445542ba-9086-4429-83a3-da3b1200f128.png)